### DISCUSSION TITLE
Top loadings

### DISCUSSION THREAD

**Discussion by hyeonjin-cho** [satijalab/seurat#4538] [Q&A] [answered]
Hi, 

I'm wondering how Seurat is finding it's top loading genes. So for PC1, I think it's 

```
head(sort((Loadings(seurat_obj)[,1])), n=30)
```

Because the list of genes that came out from the above code matched to the `VizDimLoadings()` function. 
However, for PC2 this didn't match. 

Also, where can I find the equation for loading calculations? I want to know how Seurat came up with the number assigned to each gene/PCs.

Thank you!

**Comment by timoast** [comment, marked as answer]
We use the `TopFeatures()` function to return genes for each PC to display in `VizDimLoadings()`.

The loadings are the left singular vectors, by default multiplied by the singular values: https://github.com/satijalab/seurat/blob/master/R/dimensional_reduction.R#L866
